drinks/authSystem/tests/test_login.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.models import User
+from django.test import Client
+from django.test.testcases import TestCase
+
+
+class TestLogin(TestCase):
+
+    def setUp(self):
+        self.client = Client()
```

Comment:
wydaje mi się, że `self.client` to masz od strzała i ta linijka nie jest potrzebna